Explorer/Assets/DCL/Interaction/Raycast/Tests/ExecuteRaycastSystemShould.cs

```diff
@@ -42,7 +42,7 @@ public class ExecuteRaycastSystemShould : UnitySystemTestBase<ExecuteRaycastSyst
         public void SetUp()
         {
             ISceneData sceneData = Substitute.For<ISceneData>();
-            sceneData.Geometry.Returns(new ParcelMathHelper.SceneGeometry(UnityEngine.Vector3.zero, new ParcelMathHelper.SceneCircumscribedPlanes()));
+            sceneData.Geometry.Returns(new ParcelMathHelper.SceneGeometry(UnityEngine.Vector3.zero, new ParcelMathHelper.SceneCircumscribedPlanes(), 0.0f));
```

Comment:
I suggest to avoid `UnityEngine.Vector3.zero` we could use "using UnityEngine"

Reply:
Ok. It's not related to my changes but I will do it.

Reply:
I've just realilzed that the tests use that namespace because they have to distinguish Unity's Vector3 from Protobuf's. I'm not changing it this time.